test/engine/gh-6165-multibyte-fselect.test.lua

```diff
@@ -0,0 +1,35 @@
+-- https://github.com/tarantool/tarantool/issues/6165
+-- formatted select with multibyte characters
+inspector = require('test_run').new()
+engine = inspector:get_cfg('engine')
+
+s = box.schema.space.create('test', {engine = engine, format = {{'name', 'string'}, {'surname', 'string'}}})
+_ = s:create_index('pk')
+_ = s:insert({'Ян', 'Ким'})
+s:fselect()
+s:drop()
+
+s = box.schema.space.create('test', {engine = engine, format = {{'name', 'string'}, {'surname', 'string'}}})
+_ = s:create_index('pk')
+_ = s:insert({'Абвгде', 'Жзийклмн'})
+s:fselect()
+s:drop()
+
+s = box.schema.space.create('test', {engine = engine, format = {{'Первый столбец', 'string'}, {'Второй столбец', 'string'}}})
+_ = s:create_index('pk')
+_ = s:insert({'abcdef', 'cde'})
+s:fselect()
+s:drop()
+
+s = box.schema.space.create('test', {engine = engine, format = {{'Первый столбец', 'string'}, {'Второй столбец', 'string'}}})
+_ = s:create_index('pk')
+_ = s:insert({'абв', 'гдежз'})
+s:fselect()
+s:drop()
+
+-- The problem: alignment will be still incorrect if characters have variable length
+s = box.schema.space.create('test', {engine = engine, format = {{'Abcde', 'string'}, {'Qwert', 'string'}}})
+_ = s:create_index('pk')
+_ = s:insert({'абвгд', '非常长的词'})
+s:fselect()
+s:drop()
```

Comment:
Please, add one `\n` here.
According to the [Lua style guide](https://www.tarantool.io/en/doc/latest/dev_guide/lua_style_guide/#indentation-and-formatting):
> A file should ends w/ one newline symbol, but shouldn’t ends w/ blank line (two newline symbols).